mosaicml/compute-plane
# Allow override of repository for images to avoid rate limits

For pulling images from a different repo, helm chart override may look like the following:

To override docker images:
```
mintAgent:
  image:
    repository: us-west2-docker.pkg.dev/mcloud-artifacts/mcloud/mint-agent
    tag: dev
worker:
  image:
    repository: us-west2-docker.pkg.dev/mcloud-artifacts/mcloud/worker
    tag: dev
  initImage:
    repository: us-west2-docker.pkg.dev/mcloud-artifacts/mcloud/worker-init
    tag: dev
  loggerImage:
    repository: us-west2-docker.pkg.dev/mcloud-artifacts/mcloud/mlog-agent
    tag: dev
```


For legacy 
```
mintAgent:
  image:
    repository: us-west2-docker.pkg.dev/mcloud-artifacts/mcloud/mint-agent
    tag: dev
worker:
  image:
    repository: us-west2-docker.pkg.dev/mcloud-artifacts/mcloud/worker
    tag: dev
  initImage:
    repository: ""
  loggerImage:
    repository: ""
```

## Tests 
1. described worker pod and checked that env vars were updated appropriately with no additional quotation chars 
2. check that run could be submitted after workers were helm upgraded. 
3. check for regression - ie if no overrides passed in, original docker images are pulled
